kythe/go/serving/tools/kythefs/kythefs.go

```diff
@@ -64,10 +64,10 @@ func init() {
 		"[--server SERVER_ADDRESS]")
 }
 
-type KytheFs struct {
+type kytheFs struct {
```

Comment:
Should be `kytheFS` ideally. [https://github.com/golang/go/wiki/CodeReviewComments#initialisms]

Reply:
Thank you!